neptune/new/internal/utils/deprecation.py

```diff
@@ -19,7 +19,17 @@
 
 from neptune.new.exceptions import NeptuneParametersCollision
 
-__all__ = ["deprecated", "deprecated_parameter"]
+__all__ = ["deprecated", "deprecated_parameter", "simple_warning"]
+
+
+def simple_warning(message: str, stack_level: int):
```

Comment:
Może coś z  `once` w nazwie?

Reply:
Fixed in source Pull Request -> https://github.com/neptune-ai/neptune-client/pull/1025